pkg/util/provider/machinecontroller/machine_util_test.go

```diff
@@ -2159,8 +2159,8 @@ var _ = Describe("machine_util", func() {
 					machines: []*machinev1.Machine{
 						newMachine(
 							&machinev1.MachineTemplateSpec{ObjectMeta: *newObjectMeta(&metav1.ObjectMeta{GenerateName: machineSet1Deploy1}, 0)},
-							&machinev1.MachineStatus{CurrentStatus: machinev1.CurrentStatus{Phase: machinev1.MachinePending, LastUpdateTime: metav1.NewTime(time.Now().Add(-25 * time.Minute))}},
-							nil, nil, map[string]string{machinev1.NodeLabelKey: "node-0-0"}, true, metav1.Now()),
+							&machinev1.MachineStatus{CurrentStatus: machinev1.CurrentStatus{Phase: machinev1.MachinePending}},
```

Comment:
Can you please add some new test cases?
1. Happy case: Machine is 15 mins old, and ensure machine remains in Pending
2. Case found with this issue: machine is 25 mins old, last update time is 15mins ago. Machine should move to Failed